hack/version.sh

```diff
@@ -96,7 +96,7 @@ function kubevirt::version::ldflag() {
     local key=${1}
     local val=${2}
 
-    echo "-X kubevirt.io/kubevirt/vendor/kubevirt.io/client-go/version.${key}=${val}"
+    echo "-X kubevirt.io/client-go/version.${key}=${val}"
```

Comment:
This is the wrong place to take it. You should take it from the `vendor/` directory and not use Go modules during the build.

Reply:
This would not work with 1.13(I am not sure why).
The right format is `importpath.name=string` from [what](https://tip.golang.org/cmd/link/) I know about ldflags. It should not matter that it is vendored.

Reply:
It matters because if you're vendoring, you shouldn't even have the package in `kubevirt.io/client-go/version`.

Reply:
As stated [here](https://golang.org/cmd/go/#hdr-Maintaining_module_requirements)
`If invoked with -mod=vendor, the go command loads packages from the main module's vendor directory instead of downloading modules to and loading packages from the module cache. The go command assumes the vendor directory holds correct copies of dependencies, and it does not compute the set of required module versions from go.mod files. However, the go command does check that vendor/modules.txt (generated by 'go mod vendor') contains metadata consistent with go.mod. `